apps/bfd-data-fda/src/main/java/gov/cms/bfd/data/fda/lookup/FdaDrugCodeDisplayLookup.java

```diff
@@ -83,6 +83,15 @@ public static FdaDrugCodeDisplayLookup createDrugCodeLookupForProduction() {
     return drugCodeLookupForProduction;
   }
 
+  /**
+   * Constructs a {@link FdaDrugCodeDisplayLookup} for testing purposes only.
+   *
+   * @param npiOrgMap Drug code lookup map to populate FdaDrugCodeDisplayLookup
+   */
+  public FdaDrugCodeDisplayLookup(Map<String, String> npiOrgMap) {
```

Comment:
nit: parameter name could use NDC related ? like ndcMap
same for the java doc @param

Reply:
too much copy pasta.  i'll rename

Reply:
done